susemanager-utils/susemanager-sls/modules/engines/mgr_events.py

```diff
@@ -127,13 +128,16 @@ def _insert(self, tag, data):
             fnmatch.fnmatch(tag, "suse/manager/image_deployed"),
             fnmatch.fnmatch(tag, "suse/systemid/generate")
         ]) and not self._is_salt_mine_event(tag, data) and not self._is_presence_ping(tag, data):
-            log.debug("%s: Adding event to queue -> %s", __name__, tag)
+            queue = int(hashlib.md5(data.get("id").encode(self.connection.encoding)).hexdigest()[0:8], 16) \
```

Comment:
`\` as line extenders in Python are usually considered bad style. How about splitting this in separate assignments to temporary intermediate values?

Reply:
I'm fine with that: this is generally easier to read

Reply:
:heavy_check_mark: